src/riscv/lib/src/state_backend/proof_backend/proof.rs

```diff
@@ -448,13 +501,15 @@ mod tests {
         let n4 = MerkleProof::Leaf(MerkleProofLeaf::Read(vec![123, 234, 42, 1, 2, 3]));
 
         let root = MerkleProof::Node(vec![n1.clone()]);
-        check_serialisation(root, &[(TAG_NODE << 6) | (TAG_READ << 4), 12, 15, 30, 40]);
+        check_serialisation(root, &[TAG_NODE, TAG_READ, 12, 15, 30, 40]);
 
         let root = MerkleProof::Node(vec![n1.clone(), n2.clone()]);
         check_serialisation(
             root,
             &[
-                [(TAG_NODE << 6) | (TAG_READ << 4) | (TAG_BLIND << 2)].as_ref(),
+                [TAG_NODE].as_ref(),
+                [TAG_READ].as_ref(),
+                [TAG_BLIND].as_ref(),
```

Comment:
nit, all of these can be 
```suggestion
                &[TAG_BLIND],
```